### ISSUE TITLE
Navigation grouping

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#579]
Issue group: Header
Visibility: 3
Evaluation: Partially Supports
Success Criteria: 1.3.1: Info and Relationships
Needs design: Content

### Description
In general, in addition to being placed inside an appropriately labelled `<nav>` element, the set of navigation links in menus and other navigation regions should be grouped together in a list. This provides additional context such as informing an AT user of how many navigation items to expect and how navigation lists are nested if they contain several levels.

The “Language and sign-in” navigation contains the “Your account” menu and language toggle. The “Your account” menu is contained within an unlabelled nested `<nav>`. It would be better practice to have a single `<nav>` element containing the contents as a list and nested list. For example:

``` html
<nav aria-label=“Language and sign-in”>
    <ul>
        <li>
            <button aria-expanded=“false”>Your account</button>
            <ul>…</ul>
        </li>
        <li lang=“fr-CA”>
            <a href=””>Français</a>
        </li>
    </ul>
</nav>
```

![](https://dl.airtable.com/.attachments/2d6e03b1159208b372830d431e7abc33/16a4fbe1/Capturedecranle2022-03-15a09_45_45.png)

## Related Airtable links
[Link 1](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recAYwlRcb3Q7daMG?blocks=hide)
[Link 2](https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwEUFgBd0KLqH7oo/recdyxF60Q3WUQ2hi?blocks=hide)

**Comment by amazingphilippe** [comment]
Related finding from testing:

https://airtable.com/appWwAmHwDLtpIyko/tbl38n7ofWYBuezFc/viwBYyaQxOttZ0IAw/recAYwlRcb3Q7daMG?blocks=hide

> GC Notify was difficult to navigate for participants. For a user using a screenreader, the labels weren't descriptive enough (e.g. navigation regions, homepage link). For a screen magnification user, the grey banner on the top nav was too light (but other screen magnification users didn't comment on it)

**Comment by amazingphilippe** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/579?planning-poker) with ZenHub @andrewleith

**Comment by amazingphilippe** [comment]
Relevant to #579

**Comment by amazingphilippe** [comment]
Fixed in https://github.com/cds-snc/notification-admin/pull/1301
